tests/link/pytorch/test_blockwise.py

```diff
@@ -0,0 +1,51 @@
+import numpy as np
+import pytest
+
+import pytensor
+import pytensor.tensor as pt
+from pytensor.graph.basic import Apply
+from pytensor.graph.op import Op
+from pytensor.link.pytorch.dispatch.basic import pytorch_funcify
+from pytensor.tensor.blockwise import Blockwise
+
+
+torch = pytest.importorskip("torch")
+
+
+class TestOp(Op):
+    gufunc_signature = "(m,n),(n,p)->(m,p)"
+
+    def __init__(self, final_shape):
```

Comment:
I'm missing the init call to op

Reply:
WDYM?

Reply:
I just need to add `super().__init__()` ha.